
We do not want this package to grab the xemacs binary - we use xemacs-beta
for this preview package.

  - Rene Rebe <rene@rocklinux.org>

--- xemacs-21.5.17/Makefile.in.in	2004-01-28 12:20:34.000000000 +0100
+++ xemacs-21.5.17-xft/Makefile.in.in	2004-04-28 01:17:43.000000000 +0200
@@ -395,9 +395,9 @@
 	${INSTALL_PROGRAM} src/${PROGNAME} ${bindir}/${PROGNAME}-${version}
 	-chmod 0755 ${bindir}/${PROGNAME}-${version}
 # ifdef CYGWIN
-	cd ${bindir} && $(RM) ./${PROGNAME} && ${LN_S} ${PROGNAME}-${version}.exe ./${PROGNAME}
+	cd ${bindir} && $(RM) ./${PROGNAME}-beta && ${LN_S} ${PROGNAME}-${version}.exe ./${PROGNAME}-beta
 # else
-	cd ${bindir} && $(RM) ./${PROGNAME} && ${LN_S} ${PROGNAME}-${version} ./${PROGNAME}
+	cd ${bindir} && $(RM) ./${PROGNAME}-beta && ${LN_S} ${PROGNAME}-${version} ./${PROGNAME}-beta
 # endif /* CYGWIN */
 #endif /* WIN32_NATIVE */
 	if test "${prefix}" != "${exec_prefix}"; then \
